packages/stryker/src/reporters/ProgressKeeper.ts

```diff
@@ -24,5 +26,23 @@ abstract class ProgressKeeper implements Reporter {
       this.progress.survived++;
     }
   }
+
+  protected getEtc() {
+    const timeLeft = Math.floor(this.timer.elapsedSeconds() / this.progress.tested * (this.progress.total - this.progress.tested));
+
+    if (isFinite(timeLeft)) {
+      const hours = Math.floor(timeLeft / 3600);
+      const minutes = Math.floor((timeLeft / 60) - (hours * 60));
+      const seconds = Math.floor(timeLeft - (hours * 3600) - (minutes * 60));
+
+      let output = (hours !== 0) ? `${hours}h, ` : '';
```

Comment:
I think it would be nicer to check for a positive number. If any of our inputs is negative, we will probably end up with a negative timeleft.